src/elarm_config.erl

```diff
@@ -81,25 +81,26 @@ get_unconfigured(#config_state{ missing_tab = MissingTab } = State) ->
 
 %% Get a list of all alarm_ids that have mapping configuration.
 -spec get_configured(#config_state{}) ->
-      {{ok, [{alarm_id(),alarm_config()}]}, #config_state{}} | {error, term()}.
+      {{ok, [{alarm_id(), alarm_config()}]}, #config_state{}} | {error, term()}.
 get_configured(#config_state{ config_tab = CfgTab} = State) ->
     Result = ets:tab2list(CfgTab),
     {{ok, Result}, State}.
 
 %% Get a list of all configuration data. Alarms that do not have
 %% configuration will have a default configuration.
 -spec get_all_configuration(#config_state{}) ->
-      {{ok, [{alarm_id(),alarm_config()}]}, #config_state{}} | {error, term()}.
+      {{ok, [{alarm_id(), alarm_config()}]}, #config_state{}} | {error, term()}.
 get_all_configuration(#config_state{ default_map = Default,
                                      config_tab = CfgTab,
                                      missing_tab = MissingTab } = State) ->
     Configured = ets:tab2list(CfgTab),
-    UnConfigured = [{AlarmId,Default} || {AlarmId} <- ets:tab2list(MissingTab)],
+    UnConfigured = [{AlarmId, Default}
+                    || {AlarmId} <- ets:tab2list(MissingTab)],
     Result = Configured ++ UnConfigured,
     {{ok, Result}, State}.
 
 get_default_configuration(#config_state{ default_map = Default } = State) ->
-    {{ok,Default}, State}.
+    {{ok, Default}, State}.
 
 %% Configure alarm mapping
 -spec add_configuration(alarm_id(), alarm_config(), #config_state{}) ->
```

Comment:
The spec in line 106 uses a record, please define a type for the record and use that instead.